lib/kontrast/abstract_runner.rb

```diff
@@ -0,0 +1,32 @@
+module Kontrast
+    class AbstractRunner
```

Comment:
I wouldn't call this "abstract runner", this contains image helpers so it should be called something like "ImageHelpers"